README.md

```diff
@@ -677,3 +678,195 @@ Once you have created the secrets that store the certificate info (or specified
 enableTls: true
 
 ```
+### OpenID with Starlight for Kafka
```

Comment:
Follow up work: move this stuff to a dedicated piece of documentation on the officials docs